modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -4,26 +4,103 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-. Click an asset.
-. Click Edit to add a description. The editor lets you change the appearance of text,
-insert code samples, block quotes, and horizontal lines. You can also insert a link
-to an external resource and to an image. The image has to be served on the web.
+Once an asset has been created, you can edit its portal, changing its name, its tags, its description, its icon and adding portal pages with content.
+
+image::ex2-exchange-click-on-asset.png[Illustration - Click on asset in Exchange][height=350,width=700]
+
+To enter in an asset portal, click in the asset.
+
+image::ex2-exchange-asset-portal-empty.png[Illustration - Asset Portal][height=400,width=700]
+
+This is an example of an empty asset portal.
+
+== Edit asset metadata
+
+=== Edit asset name
+
+image::ex2-exchange-click-on-edit-name.png[Illustration - Edit asset name][height=200,width=400]
+
+. Click on the pen icon near to the asset's name or press the asset name.
+. Add the text you want the name to be.
+. Press the Enter or click the check icon to change the asset name.
 +
-The Markdown editor also lets you add a link to a YouTube or Vimeo video to your asset description.
+To cancel the changes, click on the cancel icon.
++
+. This action change the name of all asset versions
+
+
+=== Edit asset description
+
+image::ex2-exchange-click-on-edit-description.png[Illustration - Edit asset description][height=200,width=400]
+
+. Click on the pen icon near to *Add description*.
+. Add the text you want the description of the asset to be.
+. Click the check icon to behind the asset description.
++
+To cancel the changes, click on the cancel icon.
++
+. This action change the description of all asset versions
+
+
+=== Edit asset icon
+
+image::ex2-exchange-click-on-edit-icon.png[Illustration - Edit asset icon][height=200,width=400]
+
+. Click on the pen icon beside the asset icon.
+. Select an image.
++
+Once the icon is uploaded, if you want to remove the icon, click on the calcel icon beside the icon
++
+. This action change the icon of all asset versions
```

Comment:
Similar to `name` and `description`, we should try to not make explicit the fact we "change the icon for all asset versions".

Reply:
Resolved.